rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+
+- A Working Group can't form until there are multiple Teams that need it to exist
+- Working Groups don't necessarily have any people allocating full-time effort to that group's goals. As a result, Working Groups are likely to push work into the OKRs of Teams, which can more easily allocate people's time.
+- Working Groups occasionally define OKRs, but they don't have to define OKRs every quarter.
+- Working Groups don’t necessarily subdivide when they get bigger. Unlike Teams, which are encouraged to form a tree of sub-teams when they grow bigger than ~5 people, Working Groups are able to convene larger groups of stakeholders.
+
+### The Core Working Group
+
+This whole structure and process relies on a Core Working Group, which acts in service of the other Teams and Working Groups to maintain a healthy, adaptable system for decision making and coordinated planning.
+
+The Core Working Group has three main functions:
+
+1. Set overall direction of the project
+2. Keep the Working Group lifecycle running smoothly
+3. Keep the RFC process running smoothly, delegating the decision making to Working Groups whenever possible
+
+### The Core Working Group is Responsible for
+- Delivering high-level, long-term strategic OKRs
+- Enforcing the lifecycle protocol for Teams and Working Groups: Guides formation of Teams/WGs, supports functioning of Teams & WGs, and dissolves them according to the lifecycle protocol
+- Keep the RFC process moving -- either delegating decision power to whichever Team/WG should make the decision or, when no other group matches the RFC, merging/rejecting the RFC themselves.
+
+### Prerogatives of the Core Working Group
+- Core team can assign/delegate ownership of a product, library, spec, problem domain to a Team or WG, So that those groups can operate with clear mandates and healthy amounts of autonomy.
+- In case of process failure or broken protocols among the teams, the Core Working Group is responsible for addressing the situation.
+- They have the authority to institute short-term measures to address pressing problems/disputes
+- They have the authority to fast-track RFCs that amend broken processes or protocols.
+
+### Membership in the Core Working Group
+
+The initial members of the Core Working Group are
+- Juan Benet (@jbenet) - BDFL
+- David Dias (@diasdavid) - Tech Lead for js-ipfs
+- Jeromy Johnson (@whyrusleeping) - Tech Lead for go-ipfs
+- Matt Zumwalt (@flyingzumwalt) - Program Manager for the IPFS ecosystem
+
+Future RFCs will define how membership of the Core Working Group will change over time. Note: For most open source orgs the "core team"/"core working group"/"steering group" is usually about 8-12 people.
+
+## Team and Working Group Lifecycle
+
+### 1. Design
+
+Anyone can propose creating a Team or Working Group. In order to be considered, it must satisfy a number of basic conditions:
+- A _Team_ **must have at least 2 people** whose **Full Time** efforts are primarily focused on that Team's work.
```

Comment:
Is this the third time this is stated in this document? It would be great if this whole thing was much smaller and got to the point sooner.